docs/index.md

```diff
@@ -11,18 +11,14 @@
 
 The easiest way to get a production grade Kubernetes cluster up and running.
 
-## 2020-05-06 etcd-manager Certificate Expiration Advisory
-
-kops versions released today contain a **critical fix** to etcd-manager: 1 year after creation (or first adopting etcd-manager), clusters will stop responding due to expiration of a TLS certificate. Upgrading kOps to 1.15.3, 1.16.2, 1.17.0-beta.2, or 1.18.0-alpha.3 is highly recommended. Please see the [advisory](./advisories/etcd-manager-certificate-expiration.md) for the full details.
-
 ## What is kOps?
 
 We like to think of it as `kubectl` for clusters.
 
 `kops` will not only help you create, destroy, upgrade and maintain production-grade, highly
 available, Kubernetes cluster, but it will also provision the necessary cloud infrastructure.
 
-AWS (Amazon Web Services) is currently officially supported, with DigitalOcean, GCE, and OpenStack in beta support, and Azure and AliCloud in alpha.
+[AWS](getting_started/aws.md) (Amazon Web Services) is currently officially supported, with [DigitalOcean](getting_started/digitalocean.md), [GCE](getting_started/gce.md), and [OpenStack](getting_started/openstack.md) in beta support, and [Azure](getting_started/azure.md) and AliCloud in alpha.
```

Comment:
Strange, GCE is feature flagged and marked as alpha in the code.

Reply:
I think it's something that we should fix and help it graduate to stable. Will add on office hours list of topics.